profile-pathway.json

```diff
@@ -14,6 +14,11 @@
         "title": "Upgrading DAML Applications",
         "description": "Learn how to upgrade running DAML applications and migrate existing data to your new model.",
         "id": "upgrading"
+      },
+      {
+        "title": "Deploying DAML on Fabric",
+        "description": "A short tutorial on deploying a full E2E DAML Application on the Fabric distributed ledger",
+        "id": "deploying-daml"
```

Comment:
I guess the course will be deploying DAML apps on different ledgers. And then Fabric is just one scenario with the course (it's fine if it stays like that for a while, we can also maybe add project:DABL). 

Will need this info to embed it on our dalm.com/learn homepage

Reply:
Yup, also revising this file slightly.